### DISCUSSION TITLE
why i need to be mark importent to  sm:!lx-p-[40px]

### DISCUSSION THREAD

**Discussion by Alihumza121** [tailwindlabs/tailwindcss#12376] [Help]
![image](https://github.com/tailwindlabs/tailwindcss/assets/48916893/077a81cd-c083-4bbb-b5ff-7f13f31eb39a)

**Comment by alex-krasikau** [comment]
@Alihumza121 can you please send the part of the Tailwind config, where you set the `screens` configuration?

**Comment by Alihumza121** [comment]
![image](https://github.com/tailwindlabs/tailwindcss/assets/48916893/3b4d68a9-0386-4948-8ee4-c0d94cdd24e2)

**Reply by alex-krasikau** [reply]
Yes, I have encountered this bug a few times. When you change the `screens` configuration, sometimes the Tailwind compiler does not correctly order the responsive class definitions in watch mode. Could you please remove the 'important', build the project, and run it in production mode to check if the issue still exists? Restarting the development server might also help with this problem.

**Reply by Alihumza121** [reply]
Yes I have tried But it was same on production as local
